src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -7,40 +7,64 @@ category: adapter
 i18nReady: true
 ---
 
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
 An SSR adapter for use with Cloudflare Pages Functions targets. Write your code in Astro/Javascript and deploy to Cloudflare Pages.
 
 ## Install
 
 Add the Cloudflare adapter to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
 
-```sh
-# Using NPM
-npx astro add cloudflare
-# Using Yarn
-yarn astro add cloudflare
-# Using PNPM
-pnpm astro add cloudflare
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npx astro add cloudflare
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm astro add cloudflare
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn astro add cloudflare
+  ```
+  </Fragment>
+</PackageManagerTabs>
 
 If you prefer to install the adapter manually instead, complete the following two steps:
 
-1. Add the Cloudflare adapter to your project's dependencies using your preferred package manager. If you’re using npm or aren’t sure, run this in the terminal:
+1. Add the Cloudflare adapter to your project's dependencies using your preferred package manager.
 
-```bash
-npm install @astrojs/cloudflare
-```
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install @astrojs/cloudflare
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add @astrojs/cloudflare
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add @astrojs/cloudflare
+  ```
+  </Fragment>
+</PackageManagerTabs>
 
 2. Add the following to your `astro.config.mjs` file:
 
-```diff lang="js"
-  // astro.config.mjs
-  import { defineConfig } from 'astro/config';
-+ import cloudflare from '@astrojs/cloudflare';
+```js title="astro.config.mjs" ins{2,5-6}
+import { defineConfig } from 'astro/config';
+import cloudflare from '@astrojs/cloudflare';
 
-  export default defineConfig({
-+   output: 'server',
-+   adapter: cloudflare(),
-  });
+export default defineConfig({
+  output: 'server',
+  adapter: cloudflare(),
+});
 ```
```

Comment:
The `ins` are not shown in the preview, any idea why?

Reply:
Its missing the `=` its `ins={2,5-6}`